packages/web-components/src/design-system-provider/index.ts

```diff
@@ -77,601 +137,879 @@ export class FluentDesignSystemProvider extends DesignSystemProvider
-  })
-  public neutralFillStealthSelectedDelta: number;
-
-  @designSystemProperty({
-    attribute: 'neutral-fill-toggle-hover-delta',
+  /**
+   * The distance from the resolved neutral fill strong color for the hover state of the neutral-fill-strong recipe.
+   *
+   * @remarks
+   * HTML attribute: neutral-fill-strong-hover-delta
+   *
+   * CSS custom property: N/A
+   */
+  @attr({
+    attribute: 'neutral-fill-strong-hover-delta',
     converter: nullableNumberConverter,
-    cssCustomProperty: false,
-    default: DesignSystemDefaults.neutralFillToggleHoverDelta,
   })
-  public neutralFillToggleHoverDelta: number;
+  @designToken(neutralFillStrongHoverDelta)
+  public neutralFillStrongHoverDelta: number;
 
-  @designSystemProperty({
-    attribute: 'neutral-fill-toggle-hover-active',
+  /**
+   * The distance from the resolved neutral fill strong color for the active state of the neutral-fill-strong recipe.
+   *
+   * @remarks
+   * HTML attribute: neutral-fill-strong-active-delta
+   *
+   * CSS custom property: N/A
+   */
+  @attr({
+    attribute: 'neutral-fill-strong-active-delta',
     converter: nullableNumberConverter,
-    cssCustomProperty: false,
-    default: DesignSystemDefaults.neutralFillToggleActiveDelta,
   })
-  public neutralFillToggleActiveDelta: number;
+  @designToken(neutralFillStrongActiveDelta)
+  public neutralFillStrongActiveDelta: number;
 
-  @designSystemProperty({
-    attribute: 'neutral-fill-toggle-hover-focus',
+  /**
+   * The distance from the resolved neutral fill strong color for the focus state of the neutral-fill-strong recipe.
+   *
+   * @remarks
+   * HTML attribute: neutral-fill-strong-focus-delta
+   *
+   * CSS custom property: N/A
+   */
+  @attr({
+    attribute: 'neutral-fill-strong-focus-delta',
     converter: nullableNumberConverter,
-    cssCustomProperty: false,
-    default: DesignSystemDefaults.neutralFillToggleFocusDelta,
   })
-  public neutralFillToggleFocusDelta: number;
+  @designToken(neutralFillStrongFocusDelta)
+  public neutralFillStrongFocusDelta: number;
 
-  @designSystemProperty({
+  /**
+   * The {@link https://www.w3.org/WAI/GL/wiki/Relative_luminance#:~:text=WCAG%20definition%20of%20relative%20luminance,and%201%20for%20lightest%20white|relative luminance} of the base layer of the application.
+   *
+   * @remarks
+   * When set to a number between 0 and 1
+   *
+   * HTML attribute: base-layer-luminance
+   *
+   * CSS custom property: N/A
+   */
+  @attr({
     attribute: 'base-layer-luminance',
     converter: nullableNumberConverter,
-    cssCustomProperty: false,
-    default: DesignSystemDefaults.baseLayerLuminance,
   })
+  @designToken(baseLayerLuminance)
   public baseLayerLuminance: number; // 0...1
 
-  @designSystemProperty({
-    attribute: 'neutral-fill-card-delta',
-    converter: nullableNumberConverter,
-    cssCustomProperty: false,
-    default: DesignSystemDefaults.neutralFillCardDelta,
-  })
-  public neutralFillCardDelta: number;
```

Comment:
We probably still want this, though it's neutralFillLayerRestDelta now.

Reply:
has that work happened here yet? I don't see that design token, and the recipe is still neutralLayerCard